modules/tensor_mechanics/include/actions/TensorMechanicsActionBase.h

```diff
@@ -20,12 +20,18 @@ class TensorMechanicsActionBase : public Action
 
   static MultiMooseEnum outputPropertiesType();
 
+  static MultiMooseEnum unrealisiticRSphericalCalls();
```

Comment:
What is this? It doesn't seem to get implemented and the name doesn't tell me what this is supposed to do.